docs/dev/connectors/pubsub.md

```diff
@@ -143,13 +143,13 @@ env.addSource(pubsubSource)
 </div>
 </div>
 
-### Atleast once guarantee
+### At least once guarantee
 
 #### SourceFunction
 
 There are several reasons why a message might be send multiple times, such as failure scenarios on Google PubSub's side.
 
-Another reason is when the acknowledgement deadline has passed. This is the time between receiving the message and between acknowledging the message. The PubSubSource will only acknowledge a message on successful checkpoints to guarantee Atleast-Once. This does mean if the time between successful checkpoints is larger than the acknowledgment deadline of your subscription messages will most likely be processed multiple times.
+Another reason is when the acknowledgement deadline has passed. This is the time between receiving the message and between acknowledging the message. The PubSubSource will only acknowledge a message on successful checkpoints to guarantee at-least-Once. This does mean if the time between successful checkpoints is larger than the acknowledgment deadline of your subscription messages will most likely be processed multiple times.
```

Comment:
I think using lower case for all three words is better: "at-least-once"

Reply:
nice catch!